sc-poc/modules/pubsub/connectors/connectors.go

```diff
@@ -0,0 +1,36 @@
+package connectors
+
+import (
+	"sync"
+
+	"github.com/ThreeDotsLabs/watermill"
+	"github.com/ThreeDotsLabs/watermill/pubsub/gochannel"
+)
+
+// Connector implements connector for pubsub module
+type Connector struct {
+	lock sync.RWMutex
+
+	// For pubsub engine
+	pubSubClient *gochannel.GoChannel
+}
+
+// New intialise the pubsub connector
+func New() (*Connector, error) {
+	pubSub := gochannel.NewGoChannel(
+		gochannel.Config{},
+		watermill.NewStdLogger(false, false),
```

Comment:
Is it possible to use caddy;s logger here?

Reply:
no the methods are not same